src/app/src/components/ClaimedFacilitiesDetails.jsx

```diff
@@ -173,9 +178,39 @@ const InputSection = ({
     );
 
     if (isSelect) {
-        const selectValue = isMultiSelect
-            ? filter(selectOptions, ({ value: option }) => includes(value, option))
-            : find(selectOptions, ['value', value]);
+        const selectValue = (() => {
+            if (!isCreatable && !isMultiSelect) {
+                return find(selectOptions, ['value', value]);
+            }
+
+            if (!isCreatable && isMultiSelect) {
+                return filter(
+                    selectOptions,
+                    ({ value: option }) => includes(value, option),
+                );
+            }
+
+            if (isCreatable && isMultiSelect) {
+                const [
+                    optionsInExistingSet,
+                    newlyCreatedOptionsPrime,
```

Comment:
Just noticed that I made a naming mistake here by leaving `Prime` on the end of this, from when I was trying out a couple ways to generate these partitions. I guess it doesn't hurt anything to leave it but if we'd like we can adjust the name to be just `newlyCreatedOptions`.